lib/whitehall_importer/edition_history.rb

```diff
@@ -91,6 +91,7 @@ def imported_entry_type(event, edition_number)
       when "rejected" then "rejected"
       when "scheduled" then "scheduled"
       when "submitted" then "submitted"
+      when "superseded" then "superseded"
```

Comment:
maybe we want to return nil here and bail out on ones that are nil rather than being specific for superseded?

Reply:
The only other case would be 'deleted'.  But I've been unable to find any examples of this to see how Whitehall presents the history.  It does not affect the NDA.

Reply:
Yeah I think if it's deleted it's no longer included in Whitehall's default scope so we don't export them. Scenario we'd hit them is if someone un-deleted something. Not sure if that's ever happened though.

Reply:
I haven't come across any cases like that.  I'll change this to return nil, then it'll be simpler to update for any future instances of this.